src/mds/MDCache.cc

```diff
@@ -177,12 +177,24 @@ MDCache::MDCache(MDSRank *m, PurgeQueue &purge_queue_) :
           check_memory_usage();
           mds->server->recall_client_state(nullptr, Server::RecallFlags::ENFORCE_MAX);
           upkeep_last_trim = clock::now();
```

Comment:
oops, this line should have been dropped. Fixing in the nautilus backport https://tracker.ceph.com/issues/50255